app/views/search/search_boxed.html.erb

```diff
@@ -57,7 +51,6 @@
   </div>
 
   <script>
-    RealtimeStatus( '#<%= id %>' );
     TrackLinks( $( '#<%= id %>' ).find( 'a' ) );
```

Comment:
Noted elsewhere but I don't think we use TrackLinks anymore. Likely worth a dedicated ticket to come back and clean it up.